### ISSUE TITLE
$smarty.capture forgets captured data from included files

### ISSUE THREAD

**Issue by d-bro** [smarty-php/smarty#235]
When upgrading from Smarty v3.1.27 to v3.1.29 I encountered the following issue:

Assume the following template files:

test-container.tpl
```html
{include 'test-header.tpl'}

{capture container}
  <p>This is captured in the container</p>
{/capture}

{include 'test-footer.tpl'}
```

test-header.tpl
```html
{capture header}
  <p>This is captured in the header</p>
{/capture}
```

test-footer.tpl
```html
{capture footer}
  <p>This captured in the footer</p>
{/capture}

{$smarty.capture.header}
{$smarty.capture.container}
{$smarty.capture.footer}
```

You would expect the following output:
```html
<p>This is captured in the header</p>
<p>This is captured in the container</p>
<p>This captured in the footer</p>
```

But the actual output is:
```html
<p>This is captured in the container</p>
<p>This captured in the footer</p>
```

For some reason, the captured data from the included file `test-header.tpl` is gone.

**Comment by monkeylogic** [comment]
I am having this issue as well.  It broke dozens of sites.  Going to try an earlier release.

**Comment by uwetews** [comment]
This was already fixed on 27.3.2016 in the master branch https://github.com/smarty-php/smarty/issues/153
